backend/config/initializers/cors.rb

```diff
@@ -3,6 +3,9 @@
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
     origins '*'
-    resource '*', headers: :any, methods: %i[get post patch put]
+    resource  '*',
+              headers: :any,
+              methods: %i[get post patch put options],
+              expose: %w[access-token expiry token-type Authorization]
```

Comment:
Needed to expose `Authorization` etc. so that the headers actually get passed back